services/health-mappings/embed_loinc_dataset.py

```diff
@@ -0,0 +1,20 @@
+import os
+from dotenv import load_dotenv
+from embeddings import health_mappings
+
+load_dotenv(override=True)
+
+def embed_loinc_dataset(input_query):
+
+    # Initialise the vector store instance
+    store = health_mappings.connect_loinc()
+
+    # Embed and uplaod data to the vector store instance
+    health_mappings.upload_loinc_data(store)
```

Comment:
I think I'd expect the logic for the upload (`upload_loinc_data`) to be in this script, so that all the upload logic is self-contained here.

If you're just using loinc as a client/consumer, you don't care - and don't need to care  - how the data is uploaded. All you need to care about is what search functionality is exposed.